package.json

```diff
@@ -49,7 +49,7 @@
     "jest-junit": "^15.0.0",
     "jest-raw-loader": "^1.0.1",
     "jest-summary-reporter": "^0.0.2",
-    "lerna": "^6.6.0",
+    "lerna": "6.6.1",
```

Comment:
It's strange that version `7.0.0` is pulled on CI.

Maybe we can use the `resolutions` field in `package.json` in case that comes from a different dependency.

Reply:
Yeah I don't understand why 7 is pulled. The CI maintenance fatigue made me pause on this this morning.

I'll try the resolutions entry, thanks!

Reply:
Or in a separate change we can update to `lerna` 7 if that can simplify things.